getting-started/display.rst

```diff
@@ -172,8 +172,7 @@ Then use the display's ``mtv`` method to view the ``calexp`` in DS9:
 
 Notice that the DS9 display is filled with colorful regions.
 These are mask regions.
-Each color reflects a different mask bit that corresponds to detections and different types of detector artifacts.
-You'll learn how to interpret these colors :ref:`later <getting-started-display-mask-colors>`, but first you'll likely want to adjust the image display.
+Each color corresponds to one of a common set of boolean flags defined for each pixel; each flag has a distinct meaning, covering a range of concepts from detector artifacts to the contribution of a pixel to a detection. In this project we call the array of these flags the "mask" or "mask plane" associated with an image. (Note: This is a distinct concept from a "mask" for an array in ``numpy``.) You'll learn :ref:`later <getting-started-display-mask-colors>` how to interpret these flags (and their colors); for now you'll likely want to adjust the image display itself.
```

Comment:
Tiny quibble: can you use "line break after every sentence" style here, as has largely been done elsewhere in the document?  It facilitates future editing and change-tracking.

Reply:
Implemented in [c49b6fd](https://github.com/lsst/pipelines_lsst_io/pull/292/commits/c49b6fd98585876cf4171f8a84d828bd87219af4).